v2.0/src/Components/Pages/ActionsPage/ActionsPage.js

```diff
@@ -265,14 +271,21 @@ class ActionsPage extends React.Component {
   };
   // renders all the actions
   renderActions(actions) {
-    if (!actions || actions.length === 0) {
+    if (!actions) {
```

Comment:
this fix is actually a subtle bug.  Javascript differs from python in this regard.
!actions actually doesnt mean the same thing in JS.  this means that as long as actions contains an object which could be an empty list. it should return that thing.  Wheras the pevious code meant that we want to check that actions is not null and is also not an empty list

Reply:
I want to distinguish between two cases passed from getContentToDisplay:
1) a community doesn't have any actions - this is when props doesn't contain any actions, and getContentToDisplay returns null;
2) the filter selection eliminates all the actions - in which case it returns an empty list.